app/views/users/show.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Here are the details of a particular user</h1>
\ No newline at end of file
+<div class="container">
+    <h1>User Detail</h1>
+    <%= render partial: "/shared/user", locals: { user: @user } %>
+    <div class="user-bio">
+        <h5>Bio</h5>
+        <p><%= @user.Bio %></p>
+    </div>
+    <div class="user-posts">
+        <h5>Posts</h5>
+        <% @top_three_posts.each.with_index(1) do |post, index| %>
+            <a href="/users<%= @user.id %>/posts<%= post.id %>">
+                <%= render partial: "/shared/post", locals: { post: post, index: index } %>
```

Comment:
- [ ] As I mentioned above, Please use `last_three_posts_for_user` method instead of `@top_three_posts` variable that you created in the `user` controller.
[Hint] You can render it like that:
```ruby
<% @user.last_three_posts_for_user.each.with_index(1) do |post, index| %>
```

Reply:
Thanks for this guidance, I will update as suggested.